pkg/lib/patternfly/patternfly-6-overrides.scss

```
@@ -163,7 +163,7 @@ $phone: 767px;
 // https://github.com/patternfly/patternfly/issues/4523
 // Override default spacing from lg -> md
 .pf-v6-l-flex {
-  gap: var(--pf-v6-l-flex--spacer-base);
+  gap: var(--pf-v6-l-flex--spacer--column--base) var(--pf-v6-l-flex--spacer--row--base);
```

Good catch! I think we'll have to revisit some other rules in this file at some point too. Gap has the order the the other way around though in this case, so would be

```suggestion
  gap: var(--pf-v6-l-flex--spacer--row--base) var(--pf-v6-l-flex--spacer--column--base);
```